code/game/objects/items/tanks/tanks.dm

```diff
@@ -318,6 +318,7 @@
 		air_contents.react(src)
 		pressure = air_contents.return_pressure()
 
+		log_atmos("[src.type] exploded with a mix of ", air_contents)
```

Comment:
You might want to also log the power of the explosion for convenience.

Reply:
Good idea